### PR TITLE
Added back console methods, to enable more functionality in the future

### PR DESCRIPTION
**Pull request by JKutscha**
As mentioned by @echebbi  in this [comment](https://github.com/pitest/pitclipse/pull/167#discussion_r681703246). I added them back now.

### CODE REVIEW

File: tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/Console.java [hunk 1]
```diff
@@ -39,4 +41,37 @@ public String getText() {
         console.show();
         return console.bot().styledText().getText();
     }
+
+    public void closeConsole() {
```
Comment (echebbi) [inline tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/Console.java:45]: We're in the `Console` class so you don't need the `Console` suffix. You can rename the method as "`close`".

File: tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/Console.java [hunk 2]
```diff
@@ -39,4 +41,37 @@ public String getText() {
...
+                view.close();
+                return;
+            }
+        }
+    }
+
+    /**
+     * If the Console view can be found, clear it
+     */
+    public void clearConsole() {
```
Comment (echebbi) [inline tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/Console.java:58]: Idem: can be named "`clear`"

File: tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/Console.java [hunk 3]
```diff
@@ -39,4 +41,37 @@ public String getText() {
...
+                if (!view.bot().styledText().getText().isEmpty()) {
+                    // use button to clear, because setting the text to "" works not synchronously
+                    view.toolbarButton("Clear Console").click();
+                }
+                return;
+            }
+        }
+    }
+
+    public SWTBotView showConsole() {
```
Comment (echebbi) [inline tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/Console.java:72]: Idem: can be named "`show`"

**Comment by LorenzoBettini** [comment]
@echebbi I'm fine with the PR, shall I merge?

**Pull request merged**